test/protocol/rest_xml.spec.coffee

```diff
@@ -214,6 +214,7 @@ describe 'AWS.Protocol.RestXml', ->
         <Error>
           <Code>InvalidArgument</Code>
           <Message>Provided param is bad</Message>
+          <Region>eu-west-1</Region>
```

Comment:
If you remove the changes from the rest_xml.js file, you won't need these tests.